.github/workflows/require_be_approval.yml

```diff
@@ -2,17 +2,15 @@ name: Require backend-review-group approval
 on:
   pull_request_review:
     types: [submitted]
-    branches:
-      - master
+    branches: master
 
 jobs:
   check-approval-requirements:
+    name: Check Approval Requirements
```

Comment:
small thing, but what do you think of changing the name of one of these to "require backend review" or something. 
I've seen people get confused about why this check hangs.

Reply:
💯